### PR TITLE
Format header image and readme.md

### PR DESCRIPTION
**Pull request by agahkarakuzu**
Small fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 
-<a href="https://qmrlab.org" target="_blank"><img src="https://github.com/qMRLab/qMRLab/blob/master/docs/logo/page_header.png?raw=true" width = "1000" height="650" align="middle"></a>
+<a href="https://qmrlab.org" target="_blank"><img src="/docs/logo/page_header.png" width = "1100" height="650" align="middle"></a>
 
-<p align="center"><h2> qMRLab is an open-source software for quantitative MR image analysis.</h2></p>
+<h2>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; qMRLab is an open-source software for quantitative MR image analysis.</h2>
```
Comment (mathieuboudreau) [inline README.md]: I'm afraid that this hack to "center" this line might not work well on mobile.

**Comment by mathieuboudreau** [comment]
@agahkarakuzu the figure is badly squished on mobile:

screenshot_20190101-151446_chrome


Is there any ways (maybe using html tags?) that this could be fixed? I checked on the SCT page, and their large image (second image) isn't squished like ours:

screenshot_20190101-151740_chrome

File: README.md (hunk 1 above)
Comment (agahkarakuzu) [reply README.md]: Resolved.

Comment (agahkarakuzu) [reply README.md]: Markdown image embed automatically preserves aspect ratio, looks good on mobile now.

**Comment by agahkarakuzu** [comment]
@mathieuboudreau I've resolved your suggestions, can you do the final check for this one?

**Pull request merged**